test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -10060,6 +10060,80 @@ public async Task RestoreNetCore_WithCustomAliases_WritesConditionWithCorrectAli
             }
         }
 
+        [Fact]
+        public async Task RestoreNetCore_CPVMProject_TransitiveDependenciesAreNotPinned()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+                var packagesForSource = new List<SimpleTestPackageContext>();
+                var packagesForProject = new List<SimpleTestPackageContext>();
+                var framework = NuGetFramework.Parse("netcoreapp2.0");
```

Comment:
```suggestion
                var framework = FrameworkConstants.CommonFrameworks.NetCoreApp20;
```

https://github.com/NuGet/NuGet.Client/blob/1f51ec51421a255c7fb83f362ee23839c0a5f631/src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs#L175

Reply:
Updated